build.gradle

```diff
@@ -110,6 +110,9 @@ dependencies {
     compile group: 'org.apache.commons', name: 'commons-math3', version: '3.5'
     compile group: 'org.apache.httpcomponents', name: 'fluent-hc', version: '4.5.2'
 
+    // Apache 2.0
+    compile group: 'com.univocity', name: 'univocity-parsers', version: '2.8.1'
```

Comment:
We use another csv library that is noted as being 1.5x slower than this one. Writing is slightly slower than Jackson, but still much faster than the supercsv we use today...might be worth switching out entirely. I can open an issue if there's no strong reason

Reply:
No reason not to use something faster, please feel free to do so!